File: Level 0/코딩 기초 트레이닝/수 조작하기 2/solution.js
const operations = {
  1: "w",
  "-1": "s",
  10: "d",
  "-10": "a",
};

function solution(numLog) {
  return numLog
    .slice(1)
    .map((num, index) => operations[num - numLog[index]])
    .join("");
}

File: Level 0/코딩 기초 트레이닝/수 조작하기 2/solution.py
operations = {
    1: "w",
    -1: "s",
    10: "d",
    -10: "a",
}


def solution(numLog):
    answer = ""

    for index in range(1, len(numLog)):
        answer += operations[numLog[index] - numLog[index - 1]]

    return answer
